lib/pushy_client/protocol_handler.rb

```diff
@@ -483,8 +483,14 @@ def self.send_signed_message(socket, method, client_private_key, session_key, me
              when :hmac_sha256
                make_header_hmac(message, session_key)
              end
-      socket.send_string(auth, ZMQ::SNDMORE)
-      socket.send_string(message)
+      begin
+        timeout(10) do
```

Comment:
```
/opt/push-jobs-client/embedded/lib/ruby/gems/2.4.0/gems/opscode-pushy-client-2.3.5/lib/pushy_client/protocol_handler.rb:487:in `send_signed_message': Object#timeout is deprecated, use Timeout.timeout instead.
```

Reply:
In case someone doesn't scroll right on that error, this should be `Timeout.timeout` instead of `timeout`.